.cspell.json

```diff
@@ -0,0 +1,131 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "languageId": "python",
+  "dictionaries": [
+    "powershell",
+    "python",
+    "go",
+    "css",
+    "html",
+    "bash",
+    "npm",
+    "softwareTerms",
+    "en_us",
+    "en-gb"
+  ],
+  "ignorePaths": [
+    "**/*.js",
+    "**/*.pyc",
+    "**/*.log",
+    "**/*.jsonl",
+    "**/*.xml",
+    "**/*.txt",
+    "scripts/docs/_build/**",
+    "src/promptflow/promptflow/azure/_restclient/flow/**",
+    "src/promptflow/tests/**",
+    "**/flow.dag.yaml"
+  ],
+  "words": [
+    "propmtflow",
+    "aoai",
+    "amlignore",
+    "mldesigner",
+    "faiss",
+    "serp",
+    "azureml",
+    "mlflow",
+    "vnet",
+    "openai",
+    "pfazure",
+    "eastus",
+    "azureai",
+    "vectordb",
+    "Qdrant",
+    "Weaviate",
+    "env",
+    "e2etests",
+    "e2etest",
+    "tablefmt",
+    "logprobs",
+    "logit",
+    "hnsw",
+    "chatml",
+    "UNLCK",
+    "KHTML",
+    "numlines",
+    "azurecr",
+    "centralus",
+    "Policheck",
+    "azuremlsdktestpypi",
+    "rediraffe",
+    "pydata",
+    "ROBOCOPY",
+    "undoc",
+    "retriable",
+    "pfutil",
+    "mgmt",
+    "wsid",
+    "westus",
+    "msrest",
+    "cref",
+    "msal"
+  ],
+  "ignoreWords": [
+    "openmpi",
```

Comment:
what's the difference between words and ignoreWords?

Reply:
Funtionally they are not so different, but `words` tends to mean domain specific word and ignore words are ... emm, other words we don't want spell to raise error.